src/test/java/Homework19CH.java

```diff
@@ -1,2 +1,23 @@
-public class Homework19CH {
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.nio.file.WatchEvent;
+
+public class Homework19CH extends BaseTest{
+
+    @Test
+    public static void DeletePlaylist() throws InterruptedException {
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example  public static void deletePlaylist()